vnext/docs/Testing.md

```diff
@@ -0,0 +1,25 @@
+# Testing ReactNative for Windows
+
+When contributing to this project, unit and integration tests should be run to help prevent new bugs and regressions.
+
+## Windows Desktop
+Tests should be run with a VSTest-compatible client
+(i.e. [VSTest.Console.exe](https://docs.microsoft.com/en-us/visualstudio/test/vstest-console-options?view=vs-2019),
+[Visual Studio Test task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/vstest?view=azure-devops)).<br/>
```

Comment:
><br/> [](start = 120, length = 5)

Can we have clean md without html syntax?

Reply:
Not if we want single line breaks.
Do you know how to render those without HTML?

Reply:
you can just place two spaces at the end of the line and that will get you a new line in markdown:
[https://stackoverflow.com/questions/26626256/how-to-insert-a-line-break-br-in-markdown](https://stackoverflow.com/questions/26626256/how-to-insert-a-line-break-br-in-markdown)

---
In reply to: [304699516](https://github.com/microsoft/react-native-windows/pull/2768#discussion_r304699516) [](ancestors = 304699516)

Reply:
Interesting! I'll try that.